### PR TITLE
Add Nop Logger for empty logger message

### PR DESCRIPTION
**Pull request by hlts2**
Signed-off-by: hlts2 <hiroto.funakoshi.hiroto@gmail.com>

### Description:

I added empty logger and test code.

#### WHY
- https://github.com/vdaas/vald/projects/3#card-58590882

### Environment:

- Go Version: 1.16
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [x] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: internal/log/logger/type.go [hunk 1]
```diff
@@ -51,6 +54,8 @@ func Atot(str string) Type {
 		return GLG
 	case "zap":
 		return ZAP
+	case "nop":
```
Comment (kpango) [inline internal/log/logger/type.go:57]:
```suggestion
	case "nop", "empty", "discard":
```

File: internal/log/nop/nop.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
+package nop
+
+import (
+	"github.com/vdaas/vald/internal/log/logger"
+)
+
+type nopLogger struct{}
```
[the remark this answers is not in the document]
Comment (hlts2) [reply internal/log/nop/nop.go:7]: I fixed

**Comment by kevindiu** [comment]
we also need to change internal/log/log.go:L46

**Comment by kpango** [comment]
@hlts2 could you please fix @kevindiu's comment?

**Comment by hlts2** [comment]
@kevindiu @kpango 
thank you for your comment.
I fixed.

**Pull request merged**